salt/states/boto_iam.py

```diff
@@ -478,8 +478,8 @@ def _delete_key(ret, access_key_id, user_name, region=None, key=None, keyid=None
         return ret
 
 
-def user_present(name, policies=None, policies_from_pillars=None, managed_policies=None, password=None, path=None,
-                 region=None, key=None, keyid=None, profile=None):
+def user_present(name, policies=None, policies_from_pillars=None, managed_policies=None, password=None,
```

Comment:
Why is the `path` option being removed here? What is the strategy for people using/expecting this argument to work?

Reply:
Almost entirely certain that was a stupid typo.  Again looking :)

Reply:
Oh, it wasn't removed - it merely folded to the next line, which doesn't appear in the snippet above.  It IS still in there however :)